package.json

```diff
@@ -113,7 +106,14 @@
   },
   "keywords": [],
   "dependencies": {
+    "angular": "1.7.9",
+    "angular-cookies": "1.7.9",
+    "angular-route": "1.7.9",
+    "cropperjs": "^1.5.6",
     "gif-frames": "^1.0.1",
-    "gifshot": "^0.4.5"
+    "gifshot": "^0.4.5",
+    "lodash": "^4.17.14",
+    "mathjs": "5.10.3",
+    "moment": "2.24.0"
```

Comment:
I have moved some of the libs here, because `devDependencies` should contain libs that are only required for development/testing.